.buildkite/pipeline.yml

```
@@ -4,7 +4,7 @@
 env:
   TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER: "Y"
   # this key is defined in local-chain bootstrap accounts list in
-  # https://github.com/serokell/serokell-profiles/blob/master/profiles/servers/jupiter/default.nix
+  # https://github.com/serokell/aquarius-infra/blob/master/servers/albali/chain.nix#L23
```

Links to the wrong like and I would remove it entirely:
```suggestion
  # https://github.com/serokell/aquarius-infra/blob/master/servers/albali/chain.nix
```
because this links to `master` and that line could change.

Alternatively linking to a specific commit would work as well, with the line fixed.